src/index.js

```diff
@@ -0,0 +1,36 @@
+import './style.css';
+import { callPage, createFooter } from './page-load';
+import homePage from './home';
+
+const container = document.getElementById('content');
+
+const createMenuTabs = () => {
+  const tabs = document.createElement('div');
+  tabs.className = 'tabsContainer';
+  tabs.innerHTML = `
+    <button class="menulink" id="home">Home</button>
+    <button class="menulink" id="menu">Our Menu</button>
+    <button class="menulink" id="contact">Contact Us</button>
```

Comment:
Let's make use of JS to generate the HTML structure.

This project requires us to use `document.createElement` and `document.appendChild` to appends HTML elements, just like you did in `line 8`.

we don't want the HTML structure to be hardcoded.

Reply:
Thanks for the suggestions offered. I follow them and remove the HTML injected code, and use exclusively DOM traversing.